src/components/Detail.jsx

```diff
@@ -1,7 +1,8 @@
+"use client";
```

Comment:
Actually we want this page-level component to be Server Component as well, but it won't work yet. we'll fix it in following episode